speculos/api/events.py

```diff
@@ -27,6 +33,14 @@ def remove_client(self, client):
 
     def broadcast(self, event):
         self.logger.debug(f"events: broadcasting {event} to ({len(self.clients)}) client(s)")
+        if self.screen_content:
+            # Reset screen content if new event is not below
+            # the last text line of current screen.
+            y_trigger = self.screen_content[-1]["y"] + MIN_LINES_HEIGHT_DISTANCE
+            if event["y"] <= y_trigger:
```

Comment:
This intermediate `y_trigger` variable is superfluous, the code is quite clear and straightforward.

Reply:
Done.